### PR TITLE
Add server push via FCM

### PR DESCRIPTION
**Pull request by aboodman**

### CODE REVIEW

**Comment by phritz** [comment]
oh, might wanna not use the default client and instead use one w a timeout eg https://github.com/rocicorp/replicache-client/blob/c8c5f4f561a4c95d7192afe9a5cf3e208e7b7d22/db/push.go#L59 otherwise if firebase hangs the todo request hangs

File: serve/util/fcm/fcm.go [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+		fmt.Println("DEBUG: Cannot poke user, no fcmKey")
+		return
+	}
+	req, err := http.NewRequest(
+		"POST",
+		"https://fcm.googleapis.com/fcm/send",
+		strings.NewReader(fmt.Sprintf(
+			`{ "data": {},  "to" : "/topics/u-%d" }`, userID)))
+	if err != nil {
+		fmt.Printf("DEBUG: Error creating request: %v\n", err)
```
Comment (arv) [inline serve/util/fcm/fcm.go:23]: [fix] missing return

Comment (aboodman) [reply serve/util/fcm/fcm.go:27]: ah, thanks.

**Comment by aboodman** [comment]
Done both, added unit tests too.

**Pull request merged**